reference/hoon-expressions/rune/bar.md

```diff
@@ -8,12 +8,78 @@ Core expressions produce cores. A core is a cell of `[battery payload]`.
 The `battery` is code, a battery of Nock formulas.  The `payload` is the data
 needed to run those formulas correctly.
 
-Four core runes (`|=`, `|.`, `|-`, and `|*`) produce a
+Five core runes (`|=`, `|.`, `|-`, `|*`, and `|$`) produce a
 core with a single arm, named `$`. We can recompute this arm
-with changes, which useful for recursion among other things:
+with changes, which is useful for recursion among other things.
 
 ## Runes
 
+### |$ "barbuc"
+
+Produces a mold builder which is a wet gate with one or more molds as its
+sample.
+
+##### Syntax
+
+Regular:
+```hoon
+|$  a=(lest term)  b=spec
+```
+
+AST:
+```hoon
+[%brbs sample=(lest term) body=spec]
+```
+
+##### Semantics
+
+`|$` is used to produce a wet gate that makes a mold builder. The mold builder
```

Comment:
```suggestion
`|$` is used to produce a wet gate that is a mold builder. The mold builder
```

...right?

Reply:
Yeah, lol. I was trying to describe the rune as a mold builder builder, but mangled the wording to make it sound like the gate is making a mold builder instead.